pkg/services/cleanup/cleanup.go

```diff
@@ -74,7 +74,7 @@ func (j cleanUpJob) String() string {
 func (srv *CleanUpService) Run(ctx context.Context) error {
 	srv.cleanUpTmpFiles(ctx)
 
-	ticker := time.NewTicker(time.Minute * 1)
+	ticker := time.NewTicker(time.Minute * 10)
```

Comment:
Lowered for HG testing, forgot to restore :upside_down_face: 
not present in any backports.